src/manager/app/services/app_data.py

```diff
@@ -0,0 +1,26 @@
+import logging
+import os
+import shutil
+
+from app.utils import check_app_name
+
+logger = logging.getLogger("uvicorn")
+RAGAPP_STATE_NAME = "ragapps"
+# The directory in the container where the state data of the ragapps is mounted.
+RAGAPPS_DIR = f"{os.getenv('STATE_DIR_LOCAL')}/{RAGAPP_STATE_NAME}"
```

Comment:
what happens if STATE_DIR_LOCAL is undefined?

Reply:
I just have an update, it will use a default folder (current_directory/data) if user is not defined